files/epub.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "epub.js",
+  "repo": "futurepress/epub.js",
+  "files": {
+    "include": ["build/epub.min.js", "build/hooks.min.js", "build/reader.min.js", "build/libs/localforage.min.js", "build/libs/zip.min.js"]
```

Comment:
@jimaek : sorry that release should have been tagged alpha, latest stable is https://github.com/futurepress/epub.js/tree/0.2.9